File: examples/happy_birthday.scala
object birthday {

  1 to 4 map { i => "Happy Birthday " + (if (i == 3) "dear Scala" else "to you") } foreach println

 (1 to 4).foldLeft("")((r,c) => r + ("\nHappy Birthday " + (if (c == 3) "dear Scala" else "to you")))
 
}

File: examples/Happy-birthday.php
<?php

function singHappyBirthday($nm) {
  for ($i = 0; $i < 4; $i++) {
    printf("Happy birthday %s!\n", (2 === $i) ? "dear {$nm}" : "to you");
  }
}
singHappyBirthday('name');